src/com/lilithsthrone/game/dialogue/npcDialogue/dominion/HarpyAttackerDialogue.java

```diff
@@ -114,13 +114,19 @@ private static String getStatus() {
 		return UtilText.parse(getHarpy(), sb.toString());
 	}
 	
-	public static final DialogueNode HARPY_ATTACKS = new DialogueNode("Assaulted!", "An angry harpy swoops down on you!", true) {
+	public static final DialogueNode HARPY_ATTACKS = new DialogueNode("Assaulted!", "An" + (Main.game.isSillyModeEnabled() ? "angery birb" : "angry harpy") + "swoops down on you!", true) {
```

Comment:
This is `static`. Will the change really work as expected?

Reply:
It has been tested and works as intended

Reply:
It has been tested and works as intended, yes.

Reply:
Regardless of whether this correctly dynamically updates, I expect people will restart the game far more often than changing the silly toggle.